public/r2/v1/swagger/swagger.json

```diff
@@ -184,42 +184,42 @@
                 }
             }
         },
-        "/namespaces/{namespaceID}/ruleset/bulk": {
+        "/namespaces/{namespaceID}/ruleset/update": {
             "post": {
                 "tags":[
                     "namespaces"
                 ],
-                "summary": "Bulk update endpoint for ruleset changes",
-                "operationId": "bulkUpdateRuleSet",
+                "summary": "Update endpoint for a ruleset, allowing for atomic changes to an entire ruleset.",
+                "operationId": "updateRuleSet",
                 "consumes": [
                     "application/json"
                 ],
                 "produces": [
                     "application/json"
                 ],
                 "parameters": [
-                    {
-                        "in": "path",
-                        "name": "namespaceID",
-                        "description": "The name of the namespace",
-                        "type": "string",
-                        "required": true
-                    },
                     {
                         "in": "body",
-                        "name": "ruleset-changes",
                         "description": "The updates to the ruleset",
                         "required": true,
                         "schema": {
-                        "$ref": "#/definitions/RuleSetChanges"
+                            "type": "object",
+                            "properties": {
+                                "rulesetChanges": {
+                                    "$ref": "#/definitions/RuleSetChanges"
+                                },
+                                "rulesetVersion": {
+                                    "type": "integer"
+                                }
+                            }
                         }
                     },
                     {
-                        "in": "body",
-                        "name": "ruleset-version",
-                        "description": "The ruleset version to atomically apply bulk updates against",
-                        "required": true,
-                        "type": "integer"
+                        "in": "path",
+                        "name": "namespaceID",
+                        "description": "The name of the namespace",
+                        "type": "string",
+                        "required": true
```

Comment:
Note this is a comment for the response, but I think for `200` response the schema for the result should be 
```
"responses": {
                    "200": {
                        "description": "The ruleset updates have been applied",
                        "schema": {
                            "$ref": "#/definitions/RuleSet"
                        }
                    },
```
Can you please fix?